templates/default/hops-system.yml.erb

```diff
@@ -54,7 +54,34 @@ data:
         "cpuRequest": "100m",
         "cpuLimit": "1",
         "memoryRequest": "100Mi",
-        "memoryLimit": "1Gi"
+        "memoryLimit": "1Gi",
+        "torchConfig": {
+          "inference_address":"http://0.0.0.0:8085",
+          "management_address":"http://0.0.0.0:8081",
+          "model_store":"/mnt/models/model-store",
+          "metrics_address":"http://0.0.0.0:8082",
+          "enable_metrics_api":true,
+          "number_of_netty_threads":4,
+          "job_queue_size":10,
+          "service_envelope":"kfserving",
+          "model_snapshot":{
+            "name":"startup.cfg",
+            "modelCount":1,
+            "models":{
+               "modelName":{
+                  "modelVersion":{
+                     "defaultVersion":true,
+                     "marName":"",
+                     "minWorkers":1,
+                     "maxWorkers":5,
+                     "batchSize":1,
+                     "maxBatchDelay":5000,
+                     "responseTimeout":120
+                  }
+               }
+            }
+          }
+        }
```

Comment:
Let's keep framework-specific configuration in a separate config map path. Something like:
```
storageInitializer: |- ...
frameworks:
    torch: |-
        inference_address=http://0.0.0.0:8085
        ...
        model_snapshot={"models": { "modelName": $MODEL_NAME, ... }}
authenticator: |- ...
```
We can add the `frameworks`  key at the same level of `authenticator`. At the moment, we will only have `torch` inside `frameworks`. Then, inside `torch` we can have the `config.properties` with the expected format. We can tell k8s to mount `frameworks.torch` as a `config.properties` file in the storage-initializer container. Then we just need to replace placeholders such as `$MODEL_NAME`, `$MODEL_VERSION` with the actual values.